utils/manifest/startLiveStream.m3u8

```diff
@@ -0,0 +1,15 @@
+#EXTM3U
+#EXT-X-START:TIME-OFFSET=0
```

Comment:
Might be good to use a positive number since 0 is often an edge case.

Reply:
I don't have the data to say otherwise, but I'm curious to know how you determined it was an edge case, since my primary use case is to use `0`...